streaming/src/test/scala/org/apache/spark/streaming/StreamingListenerSuite.scala

```diff
@@ -131,6 +132,33 @@ class StreamingListenerSuite extends TestSuiteBase with Matchers {
     }
   }
 
+  // This test is long to run an may be dependent on your machine's
+  // characteristics (high variance in estimating processing speed on a
+  // small batch)
+  ignore("latest speed reporting") {
```

Comment:
This test should not be streaminglistenersuite. Let me think about what should be right design for unit tests.